composer.json

```diff
@@ -36,11 +29,12 @@
         "symfony/validator":     "~2.8",
         "symfony/security-csrf": "~2.8",
         "symfony/twig-bridge":   "~2.8",
-        "twig/twig":             "~1.15"
+        "twig/twig":             "~1.15",
+        "ext-json":              "*"
     },
     "require-dev": {
         "jakub-onderka/php-parallel-lint": "^1.0",
-        "phpunit/phpunit": "^8.0",
+        "phpunit/phpunit": "^7.0",
```

Comment:
Why did we downgrade phpunit here?

Reply:
symfony formtype tests are failing because of strict types in phpunit 8.0, I can upgrade the components we use to 3.4, which would be out of scope for this pr, but it would solve the phpunit problems.